webapp/sources/rudder/rudder-core/src/main/resources/reportsSchema.sql

```diff
@@ -453,3 +453,20 @@ CREATE TABLE NodeFacts (
 , acceptRefuseFact  jsonb  -- the big node fact data structure
 , deleteEvent       jsonb  -- { 'date': 'rfc3339 timestamp', 'actor': 'actor name' }
 );
+
+
+Create table GlobalScore (
+  nodeId  text primary key
+, score   text
+, message text
+, details jsonb
+);
+
+Create table scoreDetails (
+  nodeId  text
+, score   text
```

Comment:
I think it could be an enumerated type, it seems to be the perfect case for that, and it would likely help regarding perf and size. But I don't know what is the migration story if for ex we want to add new values.